src/index.html

```diff
@@ -11,7 +11,135 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="formField">
+          Surname:
+          <input 
+            type="text" 
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="formField">
```

Comment:
```suggestion
        </label>
        
        <label class="formField">
```
Also, would be great to add space between one-level blocks